src/Components/Aspire.Azure.Messaging.EventHubs/EventHubsComponent.cs

```diff
@@ -89,30 +90,52 @@ protected static void EnsureConnectionStringOrNamespaceProvided(AzureMessagingEv
                 $"'ConnectionStrings:{connectionName}' or specify a 'ConnectionString' or 'FullyQualifiedNamespace' in the '{configurationSectionName}' configuration section.");
         }
 
-        // If we have a connection string, ensure there's an EntityPath if settings.EventHubName is missing
+        // Emulator: If we have a connection string, ensure there's an EntityPath if settings.EventHubName is missing
         if (!string.IsNullOrWhiteSpace(settings.ConnectionString))
         {
             // We have a connection string -- do we have an EventHubName?
             if (string.IsNullOrWhiteSpace(settings.EventHubName))
-            {
-                // look for EntityPath
+            {                
+                // look for EntityPath in the Endpoint style connection string
                 var props = EventHubsConnectionStringProperties.Parse(connectionString);
 
-                // if EntityPath is missing, throw
-                if (string.IsNullOrWhiteSpace(props.EventHubName))
+                // if EntityPath is found, capture it
+                if (!string.IsNullOrWhiteSpace(props.EventHubName))
                 {
-                    throw new InvalidOperationException(
-                        $"A {typeof(TClient).Name} could not be configured. Ensure a valid EventHubName was provided in " +
-                        $"the '{configurationSectionName}' configuration section, or include an EntityPath in the ConnectionString.");
+                    // this is used later to create the checkpoint blob container
```

Comment:
Is this comment correct? I am not sure it makes sense in the current context.

Reply:
Yeah, it is needed later to create the container for checkpoints if it doesn't already exist.